sources/Application/Player/Player.cpp

```diff
@@ -855,7 +855,7 @@ void Player::playCursorPosition(int channel) {
             }
           }
         } else {
-          NAssert(0);
+          Trace::Error("Note outside range: %02x", (unsigned int)note);
```

Comment:
I'm still getting upto speed with modern c++, but looking in n_assert.h, isn't `NAssert()` only defined for debug builds and then its a no op for release builds?

Reply:
Is that so?
I had a few crashes due to this.
You can verify by making a phrase that outputs midi notes, then transpose the phrase until the note value exceeds A9 (128) 

If you don't see this behavior, please close this as won't fix.

Reply:
@djdiskmachine  I'll double check this to be sure.

Reply:
Agree with @maks, this should not be a problem on a production build, and if it is, that's what should be fixed.

@djdiskmachine have you seen this crashes in picoTracker or in other devices?

Maybe a followup should be what to do with these cases tho. Crashing the device, debug build or not most probably doesn't make sense in any situation.

Reply:
@democloid that is a very good point! unlike on desktop, just crashing in Debug builds is not useful and makes debugging harder as you dont just get a nice core dump from the OS. So I now think this is a good change for the pico for even debug mode 👍🏻